samples/README.md

```diff
@@ -16,22 +16,23 @@ The Azure SDK repository is organized in the following folder structure, with th
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`/src` (implementation)<br>
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`/test`<br>
 
-##  Getting Started (a.k.a. `Hello World`) Samples
```

Comment:
Back ticks within titles were't rendering well in doxygen, and this doesn't add much value within the title anyway.
![image](https://user-images.githubusercontent.com/6527137/81627408-f6111d80-93b2-11ea-9468-2f1709f7d09e.png)

@danieljurek - just in case, do you know if there is a way to fix this rendering issue (low priority)?